actionmailer/lib/action_mailer/base.rb

```diff
@@ -935,6 +939,15 @@ def self.supports_path?
       false
     end
 
+    def self.class_from_string_or_symbol(klass_or_string_or_symbol)
```

Comment:
You may be right, `class_from_string_or_symbol` may also be a bit misleading in revealing that you can pass a class to it and it just returns it.